settings.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Is this file intended for commit? I would think the example is sufficient

Reply:
No, good that you spotted it!